components/TokenPage/Token.tsx

```diff
@@ -31,6 +35,18 @@ const Token = () => {
             <div className="md:!min-w-[420px]">
               {collected ? <MetaAndComments priceHidden /> : <CollectModal />}
               {!collected && isMobile && <CommentSection />}
+              {!collected && isOwner && (
+                <CollectionProvider
+                  collection={{
+                    address: token.tokenContractAddress,
+                    chainId: token.chainId || CHAIN_ID,
+                  }}
+                >
+                  <AirdropProvider>
+                    <Airdrop />
+                  </AirdropProvider>
+                </CollectionProvider>
```

Comment:
Open Closed Principle
- actual: new HTML code added inline to existing react component.
- required: new HTML component defined for new code. import the component into this existing code.